modules/api-gateway/variables.tf

```diff
@@ -55,4 +60,16 @@ variable "api_gateway_stage_name" {
 variable "vpc_endpoint_id" {
   type        = string
   description = "NGAP vpc endpoint id needed to access the api. Defaults to null."
+}
+
+variable "api_xray_tracing" {
+  type = bool
+  default = false
+  description = "Enable or Disable X-Ray Tracing on API Gateway"
+}
+
+variable "api_client_certificate_id" {
+  type = string
+  default = null
+  description = "Client Certificate ID for API Gateway"
 }
```

Comment:
This comes from KICS vulnerability API Gateway Without SSL Certificate. Setting it as null resolves the vulnerability, to set up from what I've found is in the console manually in the API Gateway -> Client Certificates -> Generate Client Certificate and then provide the certificate ID to the variable. Going to create a card to research further on client certificates.

Reply:
Sounds good.

Reply:
Note that this would go with card [ORCA-746](https://bugs.earthdata.nasa.gov/browse/ORCA-746) in that we will need to wait for NGAP guidance on TLS certificates. See the comments on securing the ALB in ORCA-713.